src/app/services/role.service.js

```diff
@@ -0,0 +1,58 @@
+angular.module('librarium').factory('IdentifyRole',
+  ['UserPersistence',  function (UserPersistence) {
+    const identifyRole = {};
+
+    identifyRole.isUser = function () {
+      let result = false;
+      const user = UserPersistence.getUserData();
+
+      if (user !== null) {
+        result = true;
+      } else {
```

Comment:
this else is unnecessary because result is already initialized with false, if user IS null it will already be false